benchmarks/run_benchmarks.py

```diff
@@ -11,13 +11,12 @@
 import os
 import subprocess
 import sys
-
+from server_util.test.server_test_util import get_arkouda_numlocales, start_arkouda_server,\
+    run_client, stop_arkouda_server
```

Comment:
For some reason, several tests would fail if I implemented this as `from server_util import *` or `from server_util.test.server_test_util import *`

Reply:
Any recommendations for changes would be appreciated if this is not being properly implemented.

Reply:
@jaketrookman - we actually tend to import only what we need to use. So, this is actually the preferred method.